tst/functions/assertions/HaveParameter.Tests.ps1

```diff
@@ -8,7 +8,7 @@ InPesterModuleScope {
             function Invoke-DummyFunction {
                 param(
                     [Parameter(Mandatory = $true)]
-                    [Alias('First')]
+                    [Alias('First', 'Another')]
```

Comment:
My only concern here is that this changes "data" of another test, would it be possible to keep those functions as they are, and introduce a new function that has two aliases.

Reply:
All tests share two functions, one with all possible options that need to be validated, and an empty one, so that was the road I continued on. 
I validated that the tests around aliases were still valid for what they were testing. I believe that actually sharing the data in this case 'enhances' the tests without having to add a number of functions to validate all kinds of scenarios. It now also validates that if `-Alias First` is tested, that it succeeds even if there are multiple aliases. 

So, my question is, is your concern a general one and was the setup of the tests perhaps not right in the first place, because they all share the same two functions here, or is there a more specific concern here that you need me to address?

Reply:
It was a general concern about sharing test data than eventually become so complicated that no-one knows what is going to break. This is not severe here so let's allow it.